src/TLuaInterpreter.cpp

```diff
@@ -12540,10 +12648,11 @@ void TLuaInterpreter::initLuaGlobals()
     //FIXME make function call in destructor lua_close(L);
 }
 
-// initialised a slimmed-down Lua state just to run the indenter in a separate sandbox
-// the indenter by default pollutes the global environment with some utility functions
-// and we don't want to tie ourselves to it by exposing them for scripting
+// Documentation: ? - internal function
 void TLuaInterpreter::initIndenterGlobals()
+// Initialised a slimmed-down Lua state just to run the indenter in a separate sandbox.
```

Comment:
I'm not sure on this style of having the comment inbetween. Typical practice is above the function like it was.

Reply:
The idea was to only have a link to documentation above function, nothing instead. Will follow your suggestion here anyway, because it's an internal function, and we have no link, yet.